spec.html

```diff
@@ -23197,6 +23196,7 @@
           1. Set the value of _O_'s [[BooleanData]] internal slot to _b_.
           1. Return _O_.
         </emu-alg>
+        <p>The `length` property of the `Boolean` constructor function is 1.</p>
```

Comment:
why is this explicit length needed? http://tc39.github.io/ecma262/#sec-boolean-constructor-boolean-value defines one required argument